editor-settings.toml

```diff
@@ -165,3 +165,12 @@
 # Type: boolean
 # Default: true
 #local =
+
+[workflow]
+
+# Filter the publishing workflows by their displayOrder value
+# min <= displayOrder <= max
+# Type: number
+# Default: undefined (unbound)
+#min =
+#max =
```

Comment:
I'm not sure if this really makes sense and would like to hear your reasoning behind this. I find this extremely unintuitive. Why should the order determine if a workflow is being shown or not? As an admin, I could see it easily happen that I want to change the order and this causes the workflow to accidentally disappear.

More than that, with the old editor being sunsetted and about to be removen, the new editor is the only place in Opencast where the tag `editor` is being used. This means that you can just tag the workflows and only the workflows you want to be shown as `editor`.

This is much more straightforward ans also means that the same functionality is not being duplicated (you are essentially trying to add a second configuration option which does the same thing as an existing one).

That means, unless you have a good argument for adding this, I really think we shouldn't add this.